src/Sylius/Bundle/AdminBundle/Resources/views/Product/Tab/_inventory.html.twig

```diff
@@ -1,18 +1 @@
-<div class="ui tab" data-tab="inventory">
-    <h3 class="ui top attached header">{{ 'sylius.ui.inventory'|trans }}</h3>
-    <div class="ui attached segment">
-        {% if product.simple %}
-            {{ form_row(form.variant.onHand) }}
-            <br />
-
-            {{ form_row(form.variant.tracked) }}
-            <div class="ui pointing above ignored label">
-                {{ form_help(form.variant.tracked) }}
-            </div>
-
-            {{ form_row(form.variant.version) }}
-        {% endif %}
-    </div>
-    
-    {{ sylius_template_event(['sylius.admin.product.' ~ action ~ '.tab_inventory', 'sylius.admin.product.tab_inventory'], {'form': form}) }}
-</div>
+{{ sylius_template_event('sylius.admin.product.tab_content', _context) }}
```

Comment:
Here will be enough to define only one template event `sylius.admin.product.tab_inventory_content` inside 
```
<div class="ui tab" data-tab="inventory">
...
</div>
```
as the menu can be customized by `ProductVariantMenuBuilderEvent`